public/docs/swagger.yaml

```diff
@@ -733,6 +753,50 @@ definitions:
       example:
         length: 17.5
         travel_image: "https://sfbaytransit.org/samtrans/route/61"
+  Create_route:
+    type: object
+    properties:
+      id:
+        type: integer
+      length:
+        type: number
+      travel_image:
+        type: string
+      created_at:
+        type: string
+      user:
+        type: object
```

Comment:
Here you can replace `object` with `'#/definitions/User'`, and not defining the User again